### PR TITLE
UX passthrough

### PR DESCRIPTION
**Pull request by brandonhundt**
- Changes how popovers are handled (`<a>` instead of `<button>`) Will now close when you click off of it.
- Changes how help text handled (using `.input-group`) which improves general layout and spacing
- Slight layout spacing (so there's more left/right margin on some of the form pages). The spacing helps the eye focus.

### CODE REVIEW

File: app/views/episodes/_form_tags.html.erb [hunk 1]
```diff
@@ -5,15 +5,10 @@
...
-        <div class="form-floating">
-          <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true} %>
-          <%= form.label :categories %>
-        </div>
-
-        <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.categories") %>">
-          <span class="material-icons">help</span>
-        </button>
+      <div class="form-floating input-group">
+        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true} %>
```
Comment (cavis) [inline app/views/episodes/_form_tags.html.erb:9]: This field still needs some work - bit buggy.  But... pre-existing condition.

**Review by cavis — APPROVED** [review]
👍 looks good to me!

File: app/views/episodes/_form_tags.html.erb (hunk 1 above)
Comment (brandonhundt) [reply app/views/episodes/_form_tags.html.erb:9]: Write a ticket!

Comment (cavis) [reply app/views/episodes/_form_tags.html.erb:9]: I think #640 covers this?  Though I disagree with that "low" priority.

**Comment by cavis** [comment]
@brandonhundt i think this needs some conflicts resolved?  But then it's good to go.

**Pull request merged**